README.md

```diff
@@ -9,6 +9,7 @@ Tested with the following apps:
 * [FreeOTP Authenticator](https://freeotp.github.io/) (open source) Availabe via [F-droid](https://f-droid.org/en/packages/org.fedorahosted.freeotp/), [Google Play](https://play.google.com/store/apps/details?id=org.fedorahosted.freeotp), and [Apple's App Store](https://itunes.apple.com/us/app/freeotp-authenticator/id872559395?mt=8)
 * [OTP Authenticator](https://github.com/0xbb/otp-authenticator) (open source) Availabe via [F-Droid](https://f-droid.org/en/packages/net.bierbaumer.otp_authenticator/) and [Google Play](https://play.google.com/store/apps/details?id=net.bierbaumer.otp_authenticator). It features a built-in QR-code reader.
 * [Google Authenticator](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2) (proprietary)
+* [KeePassXC (Linux, Windows, macOS)](https://keepassxc.org/) (open-source) Aviabilabe via [download](https://keepassxc.org/download/), package repositorys or [GitHub](http://www.github.com/keepassxreboot/keepassxc/) (Keepass also provides a plugin and Keepass2Android allows to use TOTP token)
```

Comment:
```suggestion
* [KeePassXC (Linux, Windows, macOS)](https://keepassxc.org/) (open-source) Available via [download](https://keepassxc.org/download/), package repositorys or [GitHub](http://www.github.com/keepassxreboot/keepassxc/) (Keepass also provides a plugin and Keepass2Android allows to use TOTP token)
```

Reply:
I will fix it. Should i include stuff about Keepass2Android. Is the same kdbx format. KeePass however is the original software, but runs with .NET and the library was ported. Keepass has a Plugin for TOTP. All programs store the settings and seed in protected additional attributes. 

I can use "commit suggestion" and  you squash the commits or should i amend my commit?

Reply:
Don't worry. Just pushed the fix and will integrate this now.